### PR TITLE
Improve tutorial opening and closing

### PR DESCRIPTION
**Pull request by StevenMaude**
Add some extra information based on Milan's feedback.

### CODE REVIEW

File: docs/ehrql-new-tutorial-conclusion.md [hunk 1]
```diff
@@ -62,6 +62,15 @@ as reference for writing your own dataset definitions:
 !!! todo
     Can we point to real, working Data Builder examples here?
 
+!!! tip
+    A good way to learn ehrQL is to experiment
+    by trying out ehrQL features against a simple fake dataset.
```
Comment (milanwiedemann) [inline docs/ehrql-new-tutorial-conclusion.md:67]: add link to the https://docs.opensafely.org/ehrql-reference/ here?

File: docs/ehrql-new-tutorial-conclusion.md [hunk 2]
```diff
@@ -62,6 +62,15 @@ as reference for writing your own dataset definitions:
...
+    The dataset definitions and fake data that we have provided in this tutorial
+    are a suitable starting point.
+
+    To do so, modify an example and/or the data,
+    and then [run the modified example](ehrql-new-tutorial-intro.md#running-the-tutorial-code-examples).
+
```
Comment (milanwiedemann) [inline docs/ehrql-new-tutorial-conclusion.md:73]: how about adding one simple example to guide new users, something like:

> For example, to try the function [sum_for_patient()](https://docs.opensafely.org/ehrql-reference/#421-sum-for-patient), which calculates the sum of events for each patient, you can modify the data ... and example dataset definition ...

**Review by milanwiedemann — APPROVED** [review]
great! added some ideas that might help users change the dataset definition and data

File: docs/ehrql-new-tutorial-conclusion.md (hunk 1 above)
Comment (StevenMaude) [reply docs/ehrql-new-tutorial-conclusion.md:67]: Done!

**Pull request merged**